docs/dev/docs/README.rst.rst

```diff
@@ -7,8 +7,8 @@ OCA's README
 
 * https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
 
-Our README
-----------
+IT-Projects' LLC README
+----------------------
 
 TODO:
-* https://gitlab.com/itpp/handbook
\ No newline at end of file
+* https://gitlab.com/itpp/handbook/raw/blob/master/technical-docs/README.md
```

Comment:
What it's a link for raw document?

Reply:
![image](https://user-images.githubusercontent.com/55958914/65856285-7ec9d480-e361-11e9-93d4-150e300d70ed.png)

Reply:
fixed